templates/_partials/fields/geolocation.html.twig

```diff
@@ -45,7 +45,7 @@
                         <button type="button" class="btn btn-sm btn-default expand justify-content-end"><i class="fa fa-expand"></i></button>
                         <button type="button" class="btn btn-sm btn-default compress"><i class="fa fa-compress "></i></button>
                         <div class="mapholder" style="position: relative; overflow: hidden;">
-                            <a class="editor__image--preview-image" href="/thumbs/kitten.jpg?w=1200&amp;h=1200&amp;area=files&amp;s=1e1ae532b569d01e17644e1c31b12fe9" style="background-image: url(&quot;/thumbs/kitten.jpg?w=1200&amp;h=1200&amp;area=files&amp;s=1e1ae532b569d01e17644e1c31b12fe9&quot;);"></a>
+                            <a class="editor__image--preview-image" href="/thumbs/kitten.jpg?w=1200&amp;h=1200&amp;s=1e1ae532b569d01e17644e1c31b12fe9" style="background-image: url(&quot;/thumbs/kitten.jpg?w=1200&amp;h=1200&amp;s=1e1ae532b569d01e17644e1c31b12fe9&quot;);"></a>
```

Comment:
use asset or thumbnail

Reply:
This is just a dumb placeholder. See #123 and #118